includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -804,6 +811,28 @@ public function search_button( $text = '' ) {
 		return $text;
 	}
 
+	/**
+	 * Add the sum of days taken by each student to complete a lesson with returning lesson row.
+	 *
+	 * @param array $clauses Incoming clauses.
+	 * @since  4.2.0
+	 * @return array Modified clauses.
+	 */
```

Comment:
The same docblock update could be applied here.

Reply:
We should also add an `@access` private here since technically we're only making it `public` for the filter to work, but we also don't really want anyone else to call this function directly. You can review our [deprecation policy](https://senseilms.com/documentation/deprecation-policy/) to learn why this matters.

Reply:
Thanks a lot! updated it here https://github.com/Automattic/sensei/pull/4883/commits/924df553acadf51176832236234004ca471760bf